bot/exts/fun/uwu.py

```diff
@@ -116,10 +61,57 @@ class Uwu(Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    def _word_replace(self, input_string: str) -> str:
+        """Replaces words that are keys in the word replacement hash to the values specified."""
+        for word in WORD_REPLACE_HASH:
+            input_string = input_string.replace(word, WORD_REPLACE_HASH[word])
+        return input_string
+
+    def _char_replace(self, input_string: str) -> str:
+        """Replaces characters with 'w'."""
+        return REGEX_WORD_REPLACE.sub("w", input_string)
+
+    def _stutter(self, strength: float, input_string: str) -> str:
+        """Adds stuttering to a string."""
+        return REGEX_STUTTER.sub(partial(self._stutter_replace, strength=strength), input_string, 0)
+
+    def _stutter_replace(self, match: Callable, strength: float = 0.0) -> str:
```

Comment:
Is this typing correct? A callable doesn't have a `string` attribute?

Reply:
```suggestion
    def _stutter_replace(self, match: re.Match, strength: float = 0.0) -> str:
```